Makefile

```diff
@@ -2,29 +2,29 @@
 kbs:
 	cargo build
 
-.PHONY: kbs-native-as
-kbs-native-as:
-	cargo build --no-default-features --features native-as,rustls
+.PHONY: kbs-coco-as
+kbs-coco-as:
+	cargo build --no-default-features --features coco-as-builtin,rustls
 
-.PHONY: kbs-grpc-as
-kbs-grpc-as:
-	cargo build --no-default-features --features grpc-as,rustls
+.PHONY: kbs-coco-as-grpc
+kbs-coco-as-grpc:
+	cargo build --no-default-features --features coco-as-grpc,rustls
 
-.PHONY: kbs-native-as-no-verifier
-kbs-native-as-no-verifier:
-	cargo build --no-default-features --features native-as-no-verifier,rustls
+.PHONY: kbs-coco-as-no-verifier
+kbs-coco-as-no-verifier:
+	cargo build --no-default-features --features coco-as-builtin-no-verifier,rustls
 
-.PHONY: kbs-native-as-openssl
-kbs-native-as-openssl:
-	cargo build --no-default-features --features native-as,openssl
+.PHONY: kbs-coco-as-openssl
+kbs-coco-as-openssl:
```

Comment:
I was intentionally trying to keep the Makefile target short, and since builtin is the default I though we could keep it shorter by not adding `builtin`.
But I really don't have any strong preference and can add `builtin` for consistency sake, if you prefer it this way.

Reply:
I'm also not strong with one choice. Let me do some refactor after Amber is merged. As now "built-in" is clearly for CoCo-Attestation-Service, we might use a more short feature name `"kbs-builtin-openssl"` and `"kbs-builtin"`